src/PanoViewer/PanoViewer.js

```diff
@@ -57,8 +58,19 @@ export default class PanoViewer extends Component {
 			return this;
 		}
 
+		if (!!options.image === !!options.video) {
```

Comment:
If image and video are both omitted or both specified, then trigger `INVALID_RESOURCE` error.

Reply:
Great.

There should be documentation for the new event.
like below
```
/**
* Events that is fired when animation which is triggered by inertia is ended.
* @ko 관성에 의한 애니메이션 동작이 완료되었을때 발생하는 이벤트
* @name eg.view360.PanoViewer#animationEnd
* @event
* @example
*
* viwer.on({
*	"animationEnd" : function(evt) {
*		// animation is ended.
* });
*/
```

this jsdoc is written inside of **_triggerEvent** method.

Reply:
OK, I will apply it on PanoViewer#error